### ISSUE TITLE
Clarify the use of a leading wildcard

### ISSUE THREAD

**Issue by sustr4** [publicsuffix/list#1281]
The [PSL Specification](https://publicsuffix.org/list/) says:

>  The wildcard character * (asterisk) matches any valid sequence of characters in a hostname part.

Perhaps a clarification of the term _valid sequence_ is required. I am facing an issue with my registered suffix `*.cloud.metacentrum.cz`. The DNS name of the front-end machine `cloud.metacentrum.cz` is found matching the PSL, even though only names one level deeper are supposed to match. I have had this reported, plus I have tested this myself with the C and Ruby implementations.

Is this something that could fixed by modifying the record in PSL? Or otherwise? Thank you.

**Comment by sleevi** [comment]
Dupe of https://github.com/publicsuffix/list/issues/145

**Comment by sleevi** [comment]
Er, sorry, that should have been https://github.com/publicsuffix/list/issues/694 for the more narrow case being raised here.

TL;DR: Known implementation divergence. The PSL itself hasn't been able to resolve which is "right", as both interpretations carry some risk.
